p2719.md

```diff
@@ -186,11 +236,12 @@ Delete expressions work in a way entirely analoguous to what is described above
 ```cpp
 namespace lib {
   struct Foo { };
-  void* operator new(std::type_identity<Foo>, std::size_t); // (1)
 
   struct Foo2 { };
 }
 
+void* operator new(std::size_t, std::type_identity<Foo>); // (1)
+
```

Comment:
Nope, I meant to put it in the namespace specifically because I am checking that ADL triggers.